pkg/conf/conf.go

```diff
@@ -92,6 +92,24 @@ func LoadHarvestConfig(configPath string) error {
 	return nil
 }
 
+func SafeHarvestConfig(configPath string) error {
```

Comment:
can u add some comments for this method? Also some unit test cases

Reply:
is this still needed with sha1 of cluster-uuid?

Reply:
not needed, removed.